From a19b03f8c374a4e0ad45b0b30e519ae547ab6e70 Mon Sep 17 00:00:00 2001
From: Ricardo Cerqueira <ricardo@cyngn.com>
Date: Mon, 25 Jan 2016 16:53:46 +0000
Subject: [PATCH 1/3] librilutils: Relocate pb-generated objects to their
 expected path

Relocation is less intrusive, and scales better across repositories
using these headers

Change-Id: Id11dd44bc44873a5ccef933d27e468bf916e6bdf
---
 librilutils/Android.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/librilutils/Android.mk b/librilutils/Android.mk
index 62873e9..b428f1b 100644
--- a/librilutils/Android.mk
+++ b/librilutils/Android.mk
@@ -21,6 +21,8 @@ LOCAL_PROTOC_OPTIMIZE_TYPE := nanopb-c-enable_malloc
 LOCAL_MODULE:= librilutils
 LOCAL_VENDOR_MODULE:= true
 
+LOCAL_PROTOC_OUTPUT := hardware/ril/librilutils
+
 include $(BUILD_SHARED_LIBRARY)
 
 
@@ -46,6 +48,8 @@ LOCAL_PROTOC_OPTIMIZE_TYPE := nanopb-c-enable_malloc
 LOCAL_MODULE:= librilutils_static
 LOCAL_VENDOR_MODULE:= true
 
+LOCAL_PROTOC_OUTPUT := hardware/ril/librilutils
+
 include $(BUILD_STATIC_LIBRARY)
 
 # Create java protobuf code
-- 
2.17.1

